src/Database/Dialect/MysqlSchemaDialect.php

```diff
@@ -621,3 +623,8 @@ protected function _keySql(string $prefix, array $data): string
         return $prefix . ' (' . implode(', ', $columns) . ')';
     }
 }
+
+// phpcs:disable
+// @deprecated 4.1.0 Add backwards compatible alias.
+class_alias('Cake\Database\Dialect\MysqlSchemaDialect', 'Cake\Database\Schema\MysqlSchema');
```

Comment:
This stub needs to be in ` src/Database/Schema/MysqlSchema.php` so when the old class is autoloaded composer loads the shim that sets an alias and autoloads the new class name.

Reply:
`class_alias()` in new file, and `class_exists()` in old file.

For same reasons as in 946478e7a129d4a403580f31310bbbede4817680.